src/lib/eco-router/hooks.ts

```diff
@@ -68,10 +65,8 @@ export function useEcoRouterExactIn(currencyAmountIn?: CurrencyAmount, currencyO
       .then(newTrades => {
         // Only update this invokation is not cancelled
         if (!isCancelled) {
-          batchedUpdates(() => {
```

Comment:
should we remove `batchedUpdates`?

Reply:
yes its no longer need in react18. React will batch all the state update calls

Reply:
https://reactjs.org/blog/2022/03/29/react-v18.html#new-feature-automatic-batching

Reply:
https://reactjs.org/blog/2022/03/29/react-v18.html#new-feature-automatic-batching